extension/new-tab.manifest

```diff
@@ -5,12 +5,53 @@
 # subject to an additional IP rights grant found at
 # http://polymer.github.io/PATENTS.txt
 
-import 'particles/ShowTabs/ShowTabs.manifest'
+import 'entities/WebPage.manifest'
+import 'entities/Answer.manifest'
+import 'entities/Question.manifest'
+import 'entities/VideoObject.manifest'
 
+import 'particles/ShowWebPages/ShowWebPages.manifest'
+import 'particles/ProductsFromWebPages/ProductsFromWebPages.manifest'
+
+view WebPageView of [WebPage] in 'webpages.json'
+
+# show WebPages
 recipe
-   ShowTabs
+  copy as view0
```

Comment:
if the view is only used as an input, this can be `map` rather than `copy`

Reply:
Hazarding a guess, 'map' maps a local view to a view in the arc, while 'copy' makes a local copy of a view from the arc?

Reply:
Almost, sorry, there are some notes here but still not sufficient: https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest.md#recipes

`map` maps a view external to the arc, so it would reflect any future changes to the original view, but is read only.

`copy` makes a new view, and copies the source view. So they are almost interchangeable but the latter can be read/write but will not reflect changes from the original view.